### PR TITLE
Fixes Some Tastes; Tweaks a Few

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Fixes being able to taste "metaphorical salt" and tweaks a few tastes.

Also removes serorotium, because it's a worthless Baychem that's not really makeable/findable

This also makes vitamin a subtype of nutriment, so it can better handle the new taste system and holding data on it.

Nutriment no longer restores blood

:cl: Fox McCloud
tweak: Tweaks a few reagents tastes
fix: fixes being able to taste metaphorical salt
tweak: nutriment no longer restores blood
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/disease.dm [hunk 1]
```diff
@@ -121,6 +130,7 @@
 	reagent_state = LIQUID
 	color = "#F7E6B1"
 	can_synth = FALSE
+	taste_description = "bacon"
```
Comment (Fox-McCloud) [inline code/modules/reagents/chemistry/reagents/disease.dm]: normally we don't want to be able to taste poisons, but bacon is a flavor that exists elsewhere, so to mistake this poison for something that tastes good actually makes it even stealthier.

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 2]
```diff
@@ -728,7 +729,7 @@
 	id = "mannitol"
 	description = "Mannitol is a sugar alcohol that can help alleviate cranial swelling."
 	color = "#D1D1F1"
-	taste_description = "neurogenisis"
+	taste_description = "sweetness"
```
Comment (Fox-McCloud) [inline code/modules/reagents/chemistry/reagents/medicine.dm]: mannitol is a sugar....

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 3]
```diff
@@ -786,6 +787,7 @@
 	description = "Increases run speed and eliminates stuns, can heal minor damage. If overdosed it will deal toxin damage and stun."
 	color = "#C8A5DC"
 	can_synth = FALSE
+	taste_description = "<span class='userdanger'>and unstoppable force</span>"
```
Comment (Fox-McCloud) [inline code/modules/reagents/chemistry/reagents/medicine.dm]: an not and

File: code/modules/reagents/chemistry/reagents/water.dm [hunk 4]
```diff
@@ -246,7 +246,7 @@
 	description = "Filthy water scoured from a nasty toilet bowl. Absolutely disgusting."
 	reagent_state = LIQUID
 	color = "#757547"
-	taste_description = "puke"
+	taste_description = taste_description = "the inside of a toilet... or worse"
```
Comment (Couls) [inline code/modules/reagents/chemistry/reagents/water.dm]: two taste_descriptions

Comment (Couls) [reply code/modules/reagents/chemistry/reagents/water.dm]: nevermind you caught it

**Pull request merged**